eng/pipelines/templates/steps/common.yml

```diff
@@ -6,6 +6,4 @@ steps:
     inputs:
       versionSpec: "3.6"
 
-  - template: /eng/common/pipelines/templates/steps/verify-agent-os.yml
```

Comment:
As I called out in another PR comment I think this is very important to ensure we don't end up running on machines we don't expect. We should figure out a way to verify which OS we are actually running on.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/1313